src/Export/Scripts/uniqueMods.lua

```diff
@@ -0,0 +1,57 @@
+if not table.containsId then
+	dofile("Scripts/mods.lua")
+end
+
+local itemTypes = {
+	"axe",
+	"bow",
+	"claw",
+	"dagger",
+	"mace",
+	"staff",
+	"sword",
+	"wand",
+	"helmet",
+	"body",
+	"gloves",
+	"boots",
+	"shield",
+	"quiver",
+	"amulet",
+	"ring",
+	"belt",
+	"jewel",
+	"flask",
+}
+for _, name in pairs(itemTypes) do
+	local uniqueMods = LoadModule("../Data/Uniques/Special/Uniques.lua")
+	local out = io.open("../Data/Uniques/Special/"..name..".lua", "w")
+	for line in io.lines("../Data/Uniques/"..name..".lua") do
+		local specName, specVal = line:match("^([%a ]+): (.+)$")
+		if not specName and line ~= "]],[[" then
+			local variants = line:match("{[vV]ariant:([%d,]+)}")
+			local fractured = line:match("({fractured})") or ""
+			local foundMod = false
+			local modText = line:gsub("{.+}", ""):gsub("{.+}", ""):gsub("−", "-"):lower() -- Dang EM-dash made it in here somehow
```

Comment:
pobdb.tw uses EM-DASH instead of HYPHEN-MINUS for items, so the character finds its way into PoB if we're not careful with copying new uniques.

Reply:
Ah, makes sense, I think this was only on one or two uniques, so I'll just edit those to use a hyphen instead of making this part of the script